content/changelog/2024-07-26.md

```diff
@@ -0,0 +1,31 @@
+---
+description: Better-defined compute types and improvements to app bootstrapping via CLI
+---
+
+### Clearer language around compute types
+
+Words matter. We've changed our naming convention around compute types: from `RW compute` and `RO Replica` to a cleaner, more straightforward `Primary compute` and `Read replica`.
+
+![new naming for compute types](/docs/relnotes/compute_types.png)
+
+With Neon's unique architecture, where we separate storage from compute for copy-on-write branching, you can choose the size and features for the compute that powers your branch's database independently from your data.
+
+- **Primary compute** &#8212; You don't have to add a compute to your branch right away, but once you decide to connect to your database, you need a Primary compute. Think of your primary compute as the engine that runs your branch. By adding a primary compute, you are in essence turning your branch "on". By default, your primary compute is read-write, though you can modify database permissions using traditional Postgres roles.
+- **Read replicas** &#8212; When you're ready to scale your application, you can offload read-only traffic to one or more read replicas. In Neon, read replicas work differently than traditional systems because the data is not physically replicated. Primary computes and read replicas work from the same data &#8212; a single, cost-effective source of truth.
```

Comment:
I think we could drop "a single, cost-effective source of truth". I'm not quite sure how to think about the "source of truth" part.

Reply:
Revised.